src/data/tools.js

```diff
@@ -0,0 +1,30 @@
+import { fieldMode } from '../enums'
+
+import { features } from '../config'
+
+const { CLEANUP, HARVEST, MINE, WATER } = fieldMode
+
+const tools = {
+  'watering-can': {
+    alt: 'A watering can for hydrating plants.',
+    fieldMode: WATER,
+  },
+  scythe: {
+    alt: 'A scythe for crop harvesting.',
+    fieldMode: HARVEST,
+  },
+  hoe: {
+    alt:
+      'A hoe for removing crops and disposing of them. Also returns replantable items to your inventory.',
+    fieldMode: CLEANUP,
+  },
+}
```

Comment:
I love that this has been moved out of the component! This will prevent the Object from being redefined on each render. Great call!